File: packages/midtrans_android/android/src/main/kotlin/com/deytri/midtrans_android/models/ItemDetails.kt
package com.deytri.midtrans_android.models

data class ItemDetails(
    val id: String?,
    val price: Double,
    val quantity: Int,
    val name: String,
    val brand: String?,
    val category: String?,
    val merchantName: String?,
    val url: String?,
)

File: example/ios/.symlinks/plugins/midtrans_ios/ios/Classes/models/ItemDetails.swift
struct ItemDetails {
  let id: String?
  let price: NSNumber
  let quantity: NSNumber
  let name: String
  let brand: String?
  let category: String?
  let merchantName: String?
  let url: String?
  
  init(dictionary: [String: Any]) {
    id = dictionary["id"] as? String
    price = dictionary["price"] as! NSNumber
    quantity = dictionary["quantity"] as! NSNumber
    name = dictionary["name"] as! String
    brand = dictionary["brand"] as? String
    category = dictionary["category"] as? String
    merchantName = dictionary["merchantName"] as? String
    url = dictionary["url"] as? String
  }
}
